packages/client/src/i18n/messages/views/performance.ts

```diff
@@ -51,8 +51,14 @@ const messagesToDefine = {
   },
   performanceApplicationSourcesHeader: {
     id: 'performance.reports.header.applicationSources',
-    defaultMessage: 'Sources of applications',
-    description: 'Header for sources of applications'
+    defaultMessage: 'Sources of registrations',
+    description: 'Header for sources of registrations'
+  },
+  performanceApplicationSourcesSubHeader: {
+    id: 'performance.reports.subHeader.applicationSources',
+    defaultMessage:
+      'The number and percentage of declarations started by the each system roles that have been registered',
```

Comment:
Should this be like this instead?

```suggestion
      'The number and percentage of declarations started by each system role that has been registered',
```